keylime/src/structures/capabilities_negotiation.rs

```diff
@@ -168,51 +185,158 @@ mod tests {
                     evidence_supported: vec![
                         EvidenceSupported::Certification {
                             evidence_type: "tpm_quote".to_string(),
-                            agent_capabilities: AgentCapabilities {
-                                spec_version: "2.0".to_string(),
+                            capabilities: Capabilities {
+                                component_version: "2.0".to_string(),
                                 hash_algorithms: vec!["sha3_512".to_string()],
-                                signing_schemes: vec!["rsassa".to_string()],
-                                attestation_keys: vec![
-                                    AttestationKey {
-                                        key_class: "private_key".to_string(),
-                                        key_identifier: "att_key_identifier".to_string(),
+                                signature_schemes: vec!["rsassa".to_string()],
+                                available_subjects: ShaValues {
+                                    sha1: vec![0x01, 0x02, 0x03],
+                                    sha256: vec![0x04, 0x05, 0x06],
+                                },
+                                certification_keys: vec![
+                                    CertificationKey {
+                                        key_class: "pair".to_string(),
+                                        local_identifier: "att_local_identifier".to_string(),
                                         key_algorithm: "rsa".to_string(),
-                                        public_hash: "cd293be6cea034bd45a0352775a219ef5dc7825ce55d1f7dae9762d80ce64411".to_string(),
+                                        key_size: 2048,
+                                        server_identifier: "ak".to_string(),
+                                        public: "cd293be6cea034bd45a0352775a219ef5dc7825ce55d1f7dae9762d80ce64411".to_string(),
```

Comment:
Just a detail, but this example is definitely not base64 encoded. The content here should be the public key encoded as base64.